src/test/javascript/spec/component/faq/faq-update.component.spec.ts

```
@@ -111,13 +112,38 @@
+        );
+
+        faqUpdateComponentFixture.detectChanges();
+        faqUpdateComponent.save();
+        tick();
+
+        expect(createSpy).toHaveBeenCalledExactlyOnceWith(courseId, { questionTitle: 'test1', faqState: 'PROPOSED' });
         expect(faqUpdateComponent.isSaving).toBeFalse();
     }));
 
     it('should edit a faq', fakeAsync(() => {
         activatedRoute.parent!.data = of({ course: { id: 1 }, faq: { id: 6 } });
```

Biome

[error] Forbidden non-null assertion.



(lint/style/noNonNullAssertion)